css/selectors/focus-visible-006.html

```diff
@@ -45,16 +45,15 @@
     <span id="el" contenteditable>Focus me</span>
   </div>
   <script>
-    var actions_promise;
+    setup({ explicit_done: true });
```

Comment:
Is this actually needed? Why isn't `step_func_done` enough?

Reply:
This is due to a bug in Chromium (https://crbug.com/1164600) that otherwise would detect this test as leaking.

All :focus-visible tests that have `test_driver.click()` use the same technique to avoid this issue.